### ISSUE TITLE
Error 401

### ISSUE THREAD

**Issue by leofuscaldi** [Jerrkawz/hassgalaxy#4]
Hi,

Just installed the app, entered the address and the token, next step is an error message: An Error has Occured. Status Code 401. Message: unauthorized.

What should I do?

**Comment by samhed** [comment]
I have the same issue. Got a new watch, it worked fine with the old one.

**Comment by nickjcowley** [comment]
Same issue.  Galaxy watch, url and token entered correctly, but status code: 401.  Any ideas?

**Comment by leofuscaldi** [comment]
I have formated the clock. Then, I have created a notepad, written my ha
address in one line and the token on a second line.

Installed on the clock again and run the setup o very fast, in something
like 10s. The I was successful.

Em sáb, 10 de out de 2020 08:55, nickjcowley <notifications@github.com>
escreveu:

**Comment by nickjcowley** [comment]
Okay, this error 401 for me seemed to be because the websocket had expired.  you only have around 30s to copy and paste your url and the longlived token - so you should have these copied onto a document so you can copy and paste as soon as the link is initiated by downloading the HA app to the smartwatch.  Even if the url & token are successfully transferred across to the watch, it still wont necessarily work unless you get it saved and started before the timeout.

**Comment by samhed** [comment]
Yeah, my issue was also half the time that the websocket seemed expired, and half that I failed to copy the entire token string.

**Comment by ddmmcz** [comment]
I have the same problem and Home Assistant reports: WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from ... . Did anyone manage to sign up?
